components/ConditionSetDesigner/ConditionSetItem.vue

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <div :class="['condition', {'condition--loader': !isCondition}]">
+        <template v-if="isCondition">
+            <div
+                class="condition__parameters"
+                :style="setParametersStyle">
+                <Component
```

Comment:
It would be better to make component for that and use it there - it would be more clear